### ISSUE TITLE
Separate vehicle bar keybinds?

### ISSUE THREAD

**Issue by robmonte** [Nevcairiel/Bartender4#123]
I like to use gaps in my bars as a way to organize abilities, which means half of my vehicle abilities do not have keybinds.

Is there a way to change the keybinds on a vehicle bar without changing the keys on Bar 1? Perhaps an alternative is dynamically changing keybinds for Bar 1 when entering/exiting a vehicle?

**Comment by bionicles** [comment]
hey, i found a workaround for this, but it's pretty lame, I:

- enabled bar 7 
- visibility -> disable "hide with vehicle ui" 
- state configuration -> set it to "Possess Bar" in the state
- bound keys on this bar

This enables normal function of the "possess bar" despite having random keybinds for Bar 1

**Comment by robmonte** [comment]
I don't think this is a valid workaround. The problem is I want to use my main row keys (1-6 for example) on vehicles, but also have 1-6 on my main abilities. Setting it in this way will unbind my main abilities from my Bar 1.
